pythonforandroid/recipes/python2/__init__.py

```diff
@@ -32,7 +32,9 @@ class Python2Recipe(GuestPythonRecipe):
                'patches/fix-gethostbyaddr.patch',
                'patches/fix-posix-declarations.patch',
                'patches/fix-pwd-gecos.patch',
-               'patches/fix-ctypes-util-find-library.patch']
+               'patches/fix-ctypes-util-find-library.patch',
+               'patches/fix-interpreter-version.patch',
+    ]
```

Comment:
Wait are we still spending time trying to fix Python2 related things?
Well now it's done, but I wouldn't have bothered

Reply:
> Wait are we still spending time trying to fix Python2 related things?

Well...we still support it right?...2020 is almost here...:smile: